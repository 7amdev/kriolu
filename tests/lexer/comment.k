// comment 1


// im a comment // some random text