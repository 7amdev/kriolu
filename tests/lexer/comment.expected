1 <comment> '// comment 1'
4 <comment> '// im a comment // some random text'